services/core-web/common/constants/strings.js

```diff
@@ -76,3 +76,14 @@ export const INCIDENT_FOLLOWUP_ACTIONS = {
   none: "NO",
   unknown: "HUK",
 };
+
+export const RETURN_PAGE_TYPE = {
+  LOGIN: "login",
+  SITEMINDER_LOGOUT: "smlogout",
+  LOGOUT: "logout",
+};
+
```

Comment:
Core shouldn't need this const, this is for the different auth in MineSpace

Reply:
put this back in the minespace specific strings.